src/extensions/curation/CurationSortingView/ControlPanel.tsx

```diff
@@ -0,0 +1,112 @@
+import { Grid, Paper } from '@material-ui/core';
```

Comment:
This is a repeat because the source code is synced. I guess this is a disadvantage of checking both copies into the repo. The files changed in the PRs end up being twice as big. hmmm.